### DISCUSSION TITLE
Consider removing level 70 quest gold cut

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#6115] [Suggestions]
I understand currently there is a 1/4 cut to any extra money reward a level 70 receives from completing quests at max level. (Might have been reduced to 1/2 as per Honey's input)
This is a custom change that, in my opinion, serves no other purpose than to alienate newer players and roleplayers from the server who would rather earn their coin doing side quests and having fun, than grinding primals and employing bots to run the auction house.

It also feels completely unnecessary, partly because the level 70 gold rewards are broken to begin with and only award 10% of the intended money (bug reported here: https://github.com/chromiecraft/chromiecraft/issues/6114) and partly because the level 70 quest xp values are comparatively much smaller than for example level 30 ones. While level 30 or 40 was never supposed to be a level cap that is converted to money, and as such the exp values weren't balanced for the known formula and could provide imbalanced amount of money with an arbitrary progression cap, this is no longer the case on lvl 70.

Quests during TBC were known to award 5-6 gold on top of any other rewards from the exp conversion alone, and saving up all quests in Outland to complete at max level was a profitable and popular way of farming money for fresh players and alts alike. Multiple anecdotal sources confirm that this could easily yield a total of 5-6k gold, which is completely intended and blizzlike, and since it's a once-off amount, it mostly impacts the newer players without inflating the entire economy.

I understand the level 70 cap and the slow progression on CC creates unique circumstances and tuning (including any gold rewards) is sometimes necessary to keep the game balanced, but I don't see any benefit to the current system which cuts money rewards exclusively for quests.
If anything, the population should be encouraged to do quests, not discouraged from it.

**Comment by Squirrelznut** [comment]
Agreed.
As an RPER who only has a couple 70s and dislikes farming, it feels like I'll NEVER have the gold for fast flying, plus due to the low wealth of the server, a lot of mats are sold at such low prices that if you have to put it up twice on AH for 24h, you already lost money compared to just vendoring.
SOME inflation is desirable imho.
